enterprise/frontend/src/metabase-enterprise/whitelabel/components/IllustrationDescription.tsx

```diff
@@ -0,0 +1,75 @@
+import { t } from "ttag";
+
+import { Flex, Icon, Stack, Text, HoverCard } from "metabase/ui";
+
+type IllustrationType =
+  | "login-page"
```

Comment:
Not a fan of the component having this many variants that'll basically "switch" the content over, but I saw that you changed it in the next PR so 👍

Reply:
oh yeah, it's been simplified to either `background` or `icon` which is think is the right amount of abstraction at this point. Let's see if that needs changing over time.